sdk/src/utils/xmp_inmemory_utils.rs

```diff
@@ -53,113 +55,124 @@ impl XmpInfo {
     }
 }
 
-/// Extract an a value from XMP using a key
+/// Extract an a value from XMP using a key.
```

Comment:
Still has a typo.  Should be "Extract aa value"

Reply:
fixed